### PR TITLE
[1.x] Fix CI `sbt: command not found`

### PR DESCRIPTION
**Pull request by Friendseeker**
[Jobs](https://github.com/sbt/zinc/actions/runs/11260318815/job/31311247166) are failing with message

> bin/run-ci.sh: line 5: sbt: command not found

This PR manually installs sbt to address the issue.

### CODE REVIEW

**Comment by Friendseeker** [comment]
Guess Github decided to nuke sbt from their runner images....

----

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md

Indeed, they did....

File: .github/workflows/dependency-graph.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Submit Dependency Graph
 on:
   push:
-    branches: [1.10.x] # default branch of the project
+    branches: [1.10.x, develop] # default branch of the project
```
Comment (jtjeferreira) [inline .github/workflows/dependency-graph.yml:5]: why is this needed?

**Review by jtjeferreira — APPROVED** [review]

File: .github/workflows/dependency-graph.yml (hunk 1 above)
Comment (Friendseeker) [reply .github/workflows/dependency-graph.yml:5]: This is not strictly needed but Eugene recommended it when reviewing a similar PR

https://github.com/sbt/sbt/pull/7746#pullrequestreview-2357811045

I guess the motivation is that we can use the same `dependency-graph.yml` across both 1.10.x and develop branch.

**Review by SethTisue — APPROVED** [review]

**Review by eed3si9n — APPROVED** [review]
